statsmodels/tsa/automatic/tests/test_forecast.py

```diff
@@ -0,0 +1,56 @@
+"""unit test for automatic sarimax forecasting."""
+import os
+import pytest
+import numpy as np
+import pandas as pd
+import statsmodels.api as sm
+from statsmodels import datasets
+from numpy.testing import assert_equal, assert_allclose
+
+from statsmodels.tsa.automatic import Forecast
+from statsmodels.tsa.automatic import ForecastSet
+
+current_path = os.path.dirname(os.path.abspath(__file__))
+macrodata = datasets.macrodata.load_pandas().data
+macrodata.index = pd.PeriodIndex(start='1959Q1', end='2009Q3', freq='Q')
+
+
+def test_forecast():
```

Comment:
Can you change this to `test_forecast_smoke` and add a comment like (just `# smoke test` is enough) so that we can easily identify that these weren't tested against another package?

Reply:
I'll do this, and push a commit.